extensions/trace_propagators/src/test/java/io/opentelemetry/extensions/trace/propagation/TraceMultiPropagatorTest.java

```diff
@@ -76,7 +76,20 @@ void fields() {
 
     List<String> fields = prop.fields();
     assertThat(fields).hasSize(4);
```

Comment:
Nit: we could skip this since it is already covered by `containsOnly`
The same at line 91

Reply:
no, because `containsOnly` doesn't check duplicates.
e.g. `assertThat(Lists.newArrayList("one", "one")).containsOnly("one");` will pass.

Reply:
there is `containsExactlyInAnyOrder(...)` which can be used to skip checking size

Reply:
done

Reply:
Thx for the clarification! The semantics of the name is misleading :stuck_out_tongue_closed_eyes: